e2e/cypress/support/planning/advancedSearch.ts

```diff
@@ -37,15 +37,15 @@ export class AdvancedSearch {
             state: new SelectMetaTerms(getSearchPanel, '[data-test-id=field-state]'),
             only_posted: new ToggleInput(
                 getSearchPanel,
-                '[data-test-id=field-posted] .sd-toggle'
+                '[data-test-id=field-posted] .sd-switch__wrapper'
```

Comment:
Merging `develop` should fix these e2e issues (https://github.com/superdesk/superdesk-planning/pull/1605)